ext/bg/js/database.js

```diff
@@ -585,7 +585,7 @@ class Database {
             tagDataLoaded(summary, bank, ++bankTotalCount, bankLoadedCount++);
         }
 
-        const loadBank = async (summary, namer, count, callback) => {
+        const loadBank = async (namer, count, callback) => {
```

Comment:
Is there a rule that allows specifying variables used from the parent scope like the `/*global name1, name2*/`, similar to [PHP anonymous function](https://www.php.net/manual/en/functions.anonymous.php) `use`?

Reply:
None that I'm aware of, and I think it would also be overkill. Closure capture is such a fundamental concept that I don't think it should be explicitly stated; this must be done in C++ and is honestly quite tiresome. IDEs will often also be able to highlight uses of a variable, so this can also provide clarity about variable use.

Reply:
I also thought the same when I started doing PHP last year, but now I'm used to it. Most programming languages seem to capture automatically, even Rust.